README.md

```diff
@@ -79,13 +79,7 @@ logger.log("What we've got here is...failure to communicate", "Some men you just
 );
 ```
 
-Flush all log messages and close down:
-```javascript
-logger.close(function(){
-    console.log('All done - cookie now?');
-    process.exit();
-});
-```
+The connection will be closed automatically at the end of your program.
```

Comment:
The [docs](https://nodejs.org/api/dgram.html) say:
> Calling `unref` on a socket will allow the program to exit if this is the only
> active socket in the event system [...]

...so I _think_ that the socket remains active until the program exits, but I suppose it would be functionally identical if it were closed as soon as it was GC'd